app/src/main/java/com/example/cinemapp/ui/splash/SplashViewModel.kt

```diff
@@ -31,4 +36,24 @@ class SplashViewModel(
             _goToNextScreen.emit(validSession)
         }
     }
+
+    fun setupUserPrefs() {
+        setTheme()
+        setLanguage()
+    }
+
+    private fun setLanguage() {
+        viewModelScope.launch {
+            if (userPrefs.getLanguage().firstOrNull() == null) {
+                userPrefs.setLanguage(null)
```

Comment:
If the selected language is null or language is not selected, set it to null again?

Reply:
Passing null to the function will cause it to set the default language, but it can be changed to have a default argument value and accept just setLanguage()